public_html/wp-content/themes/wporg-events-2023/package.json

```
@@ -12,6 +12,7 @@
 		"postcss-preset-env": "^9.3.0"
 	},
 	"scripts": {
+		"build:block": "wp-scripts build",
```

```suggestion
		"build": "wp-scripts build && postcss postcss/style.pcss postcss/editor.pcss --dir . --ext css",
```

_(then remove the existing `build` command below)_

This way we don't have to remember 2 different commands.

---

Yeah, that makes more sense. I rammed that in for the prototype.

---

done in 855fa570